packages/smooth_app/lib/pages/product_page.dart

```diff
@@ -328,6 +329,16 @@ class _ProductPageState extends State<ProductPage> {
         userPreferencesModel.getOrderedVariables(userPreferences);
     final List<Widget> listItems = <Widget>[];
 
+    if (_product.imgSmallUrl != null) {
```

Comment:
I made imgSmallUrl deprecated to better match the names from the OFF API, there is now imageFrontUrl and imageFrontSmallUrl. Note that the "small" image is indeed small: 200px height, whereas the non-small is 400px. In the product page, it probably makes sense to show the larger.

But we can rethink this later, there are probably other changes related to product images urls / sizes.

Reply:
ah, I forgot it's not in the currently released openfoodfacts package, you can ignore my comment :)